download the app's skeleton as a .zip file link updated

anupsati opened the pull request:
download the app's skeleton as a .zip file link was pointing wrong url. It's updated with correct link.

azurecla commented:
Hi __@aranm__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

anupsati committed:
download the app's skeleton as a .zip file link updated

download the app's skeleton as a .zip file link was pointing wrong url. It's updated
